sdk/communication/azure-communication-administration/src/samples/java/com/azure/communication/administration/ReadmeSamples.java

```diff
@@ -141,6 +146,28 @@ public PhoneNumberClient createPhoneNumberClient() {
         return phoneNumberClient;
     }
 
+    /**
+     * Sample code for creating an async Phone Number Client.
+     *
+     * @return the Phone Number Async Client.
+     */
+    public PhoneNumberAsyncClient createPhoneNumberAsyncClient() {
+        // You can find your endpoint and access token from your resource in the Azure Portal
+        String endpoint = "https://<RESOURCE_NAME>.communication.azure.com";
+        String accessKey = "SECRET";
+
+        // Create an HttpClient builder of your choice and customize it
+        HttpClient httpClient = new NettyAsyncHttpClientBuilder().build();
+
+        PhoneNumberAsyncClient phoneNumberClient = new PhoneNumberClientBuilder()
+            .endpoint(endpoint)
+            .accessKey(accessKey)
+            .httpClient(httpClient)
+            .buildAsyncClient();
+
+        return phoneNumberClient;
+    }
```

Comment:
Wondering if really need to have the async client sample here? Maybe just have the sync sample (using syncPoller) and that should be unless you see a value in doing both?